index.php

```diff
@@ -3,11 +3,19 @@
 require_once 'src/connectToDb.php';
 require_once 'src/Models/BlogModel.php';
 
+session_start();
 $db = connectToDb();
-
 $blogModel = new BlogModel($db);
 
 $blogs = $blogModel->getAllPosts();
+
+if ($_SESSION['userid'] === false){
```

Comment:
I suspect you are going to be checking this a lot.

It would be worth thinking about some kind of class you could build that could take care of the various different things you're going to need to do with the session.

For example

Logging someone in
Checking if someone is logged in